docs/using_helm.md

```diff
@@ -211,11 +211,14 @@ imageTag: 10.1.14-r3
 # mariadbDatabase:
 ```
 
-You can then override any of these settings in a YAML formatted file,
+You can then override any of these settings in a JSON or YAML formatted file,
```

Comment:
"JSON or" should be removed based on previous feedback (-f doesn't handle JSON files)